asserts/snap_asserts.go

```diff
@@ -316,3 +316,134 @@ func assembleSnapRevision(assert assertionBase) (Assertion, error) {
 		timestamp:     timestamp,
 	}, nil
 }
+
+// SnapValidation holds a validation assertion, describing that a combination of
+// (snap-id, snap-revision, approved-snap-id, approved-revision, series)
+// has been validated by testing.
```

Comment:
A few things here:

* Remove the empty line after between the comment and the type so they're attached.
* The type is called just `Validation`, not `SnapValidation`.
* I don't think `snap-revision` should be listed.
* I'd either list `series` at the start (to match the primary key) or maybe move it out completely, e.g. "(snap-id, approved-snap-id, approved-revision) has been validated by testing, for the series". Although that language is not idea, but then ...
* What does "validated by testing" mean? Perhaps it would be better to explain what the assertion is used for instead?